CS/XPO/Readme.md

```diff
@@ -12,7 +12,7 @@ This is a variation of the [How to generate and assign a sequential number for
 
```

Comment:
The link title on the line above (line 11) is not the same as the actual ticket title. Can you update the link title as follows?

`This is a variation of the [How to generate a sequential number for a business object within a database transaction](https://www.devexpress.com/Support/Center/p/E2620) XPO example, which was specially adapted for XAF applications.`

Reply:
Done